CASM-4860 Add a Note to check latest version of docs-csm is installed or not, if Upgrading CSM manually (#5377)
Cray-HPE/docs-csm: operations/iuf/workflows/preparation.md

   [latest version of `docs-csm`](https://github.com/Cray-HPE/docs-csm/blob/release/1.6/update_product_stream/README.md#check-for-latest-documentation)
    is installed for the target CSM version being installed or upgraded.

>     **`NOTE`** This subsection is optional and can be skipped if upgrading CSM through IUF.

Reviewer: ```suggestion
    **`NOTE`** When using IUF to upgrade CSM, please skip this subsection.
```
